sdk/textanalytics/tests.yml

```diff
@@ -10,12 +10,13 @@ stages:
           SubscriptionConfigurations:
             - $(sub-config-azure-cloud-test-resources)
             - $(sub-config-text-analytics-azure-cloud-test-resources)
+          MatrixReplace:
+            - TestSamples=.*/true
         UsGov:
           SubscriptionConfiguration: $(sub-config-gov-test-resources)
         China:
           SubscriptionConfiguration: $(sub-config-cn-test-resources)
-      MatrixReplace:
-        - TestSamples=.*/true
+      SupportedClouds: 'Public,UsGov,China'
```

Comment:
Will this only affect the weekly pipeline or will our regular live pipeline also run in all clouds?

Reply:
@kristapratico  Only affect the weekly pipeline.